Mathlib/CategoryTheory/GuitartExact/KanExtension.lean

```diff
@@ -137,6 +146,54 @@ lemma hasLeftKanExtension [w.GuitartExact]
   have := w.hasPointwiseLeftKanExtension F
   infer_instance
 
+section
+
+open Functor
+
+section
+
+variable [∀ (F : C₁ ⥤ D), L.HasLeftKanExtension F] [∀ (F : C₂ ⥤ D), R.HasLeftKanExtension F]
+
+/-- The base change natural transformation for left Kan extensions associated to
+a 2-square. -/
+@[simps -isSimp]
+noncomputable def lanBaseChange :
+    (whiskeringLeft C₁ C₂ D).obj T ⋙ L.lan ⟶ R.lan ⋙ (whiskeringLeft C₃ C₄ D).obj B where
```

Comment:
Would it make sense to phrase this as a `TwoSquare` as well? (Genuine question; you know the intended usage of this better than I do! :) )
```suggestion
    TwoSquare ((whiskeringLeft C₁ C₂ D).obj T) R.lan L.lan ((whiskeringLeft C₃ C₄ D).obj B) where
```